js/core/src/Blocks.svelte

```diff
@@ -391,6 +406,9 @@
 			submit_map.set(dep_index, submission);
 
 			for await (const message of submission) {
+				if (payload.js_implementation) {
```

Comment:
If a function has a js_implementation, we still want to make the API call to the backend since it might have downstream effects. For example, updating the "type" of an "Image" could affect the processing of a subsequent call.

Reply:
So it runs both on the frontend and backend?

Reply:
Yes, maybe we can be smarter here, but for now yes. You immediately see the results in the frontend, which is the improvement